scripts/regenerate-riot-latest.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -e
+
+DDTEST_CMD=scripts/ddtest
+
+pkgs=$(python scripts/freshvenvs.py | cut -d':' -f1)
+echo $pkgs
+
+if ! $DDTEST_CMD; then
+    echo "Command '$DDTEST_CMD' failed."
+    exit 1
+fi
+
+for pkg in ${pkgs[*]}; do
+    echo "Checking if new latest version exists for $pkg"
+    export VENV_NAME="$pkg"
+    RIOT_HASHES=( $(riot list --hash-only "^${VENV_NAME}$") )
```

Comment:
This seems to rely on the assumption that the name of an integrated package will always also be the name of the relevant riot environment, which is not always the case. https://github.com/DataDog/dd-trace-py/blob/e94ed6049e2addab48d51fbbc1404f3995a3c941/ddtrace/_monkey.py#L115